### PR TITLE
Refactor rake task to make it possible to run jetty:config tasks from root of working directory

### LINKED ISSUE #69: Make it possible to run jetty:config tasks from root of working directory
see #59 for some history

### PR DESCRIPTION
**Pull request by jpstroop**
This should work now (i.e., as it now says in the README):

```
    $ rake engine_cart:generate
    $ rake jetty:clean
    $ rake curation_concerns:jetty:config
    $ rake spec
```

Need  to see it on Travis, and someone else pulling it down, running the above + the specs wouldn't hurt either.

Closes #69

### CODE REVIEW

**Comment by kevinreiss** [comment]
:+1: This checked out locally for me too @jpstroop. Going to merge.

File: spec/test_app_templates/lib/generators/test_app_generator.rb [hunk 1]
```diff
@@ -6,8 +6,13 @@ class TestAppGenerator < Rails::Generators::Base
   def install_engine
     generate 'curation_concerns:install'
   end
+  
+  def comment_out_web_console
+    gsub_file "Gemfile", "gem 'web-console'", "# gem 'web-console'"
+  end
```
Comment (kevinreiss) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: Is this something we still need with the fix to engine_cart?

Comment (jpstroop) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: bah...thought I took that out...

Comment (jpstroop) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: better let it run once more :watch:

**Pull request merged**